main.cu

```diff
@@ -130,40 +130,36 @@ __global__ void RpearsonCorr_gpu_atomic_float_same_block(
             atomicAdd(x_norm + col2_num * m + col1_num, x_element_norm);
             atomicAdd(y_norm + col1_num * m + col2_num, y_element_norm);
             atomicAdd(y_norm + col2_num * m + col1_num, y_element_norm);
+	
           }
       }
   }
 }
-__global__ void RpearsonCorrn_gpu_atomic_float_different_blocks(float* array, float* array2, const int n, const int m, const int m_b, float* result ){
-	int row = blockIdx.y*blockDim.y +threadIdx.y;
-	int row2 = blockIdx.x*blockDim.x +threadIdx.x;
-	float epsilon=0.01;
-	for (int col1_num=0;col1_num<m;++col1_num){
-	float num = 0;
-	float sum1 = 0;
-	float sum2 = 0;
-	float dist = 0;
+
+__global__ void RpearsonCorr3_gpu_atomic_float_different_blocks(float* array, float* array2, const int n, const int m, const int m_b, float* r1,float *r2, float* r3 ){

```

Comment:
Please, name variables not as `r1`, `r2` and `r3`

Reply:
ok